src/main/java/mate/academy/dao/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
```

Comment:
The database URL should not be left incomplete. Ensure that the `DB_URL` variable contains the full JDBC URL to connect to the database.